tooling/nargo_cli/build.rs

```diff
@@ -198,8 +198,8 @@ fn test_{test_name}(force_brillig: ForceBrillig, inliner_aggressiveness: Inliner
         nargo.arg("--force-brillig");
 
         // Set the maximum increase so that part of the optimization is exercised (it might fail).
-        nargo.arg("--max-bytecode-increase-percent");
-        nargo.arg("50");
+        // nargo.arg("--max-bytecode-increase-percent");
```

Comment:
Noting that as we unroll more loops, this is potentially going to lead to panics. When we have a non-aggressive inliner the compiler is not going to have yet resolved certain intrinsics which we expect to be entirely known at compile-time (e.g. DerivePedersenGenerators).